Update Homepage to align with DS starter updates

KiriHoy opened the pull request:
The homepage on the DS starter was updated. Chameleon needed to be updated to reflect this.

sukhrajghuman commented:
@KiriHoy awesome!

can we also run the following commands to get the latest versions of some of our compenents

```
rm -rf node_modules/
npm i
```
```

KiriHoy commented:
> @KiriHoy awesome!
> 
> can we also run the following commands to get the latest versions of some of our compenents
> 
> ```
> rm -rf node_modules/
> npm i
> ```
> 
> Also we probably have this problem in here as well ?
> 
> [govau/design-system-starter#157](https://github.com/govau/design-system-starter/issues/157)

@sukhrajghuman The commands have been run. [govau/design-system-starter#157](https://github.com/govau/design-system-starter/issues/157) shouldn't be an issue, I carried everything across as it was in the starter.  So the header is sitting inside the correct `col-md` now, I fixed it when I updated the cta 😄

sukhrajghuman commented:
okay awesome!! merge!
